src/util/codec/mysql/json.rs

```diff
@@ -563,4 +696,36 @@ mod test {
             assert!(resp.is_err());
         }
     }
+
+    #[test]
+    fn test_cmp_json() {
+        let json_null = Json::from_str("null").unwrap();
+        let json_bool_true = Json::from_str("true").unwrap();
+        let json_bool_false = Json::from_str("false").unwrap();
+        let json_integer_big = Json::from_str("5").unwrap();
+        let json_integer_small = Json::from_str("3").unwrap();
+        let json_double_big = Json::from_str("5.0").unwrap();
+        let json_double_small = Json::from_str("3.1").unwrap();
+        let json_str_big = Json::from_str(r#""hello, world""#).unwrap();
+        let json_str_small = Json::from_str(r#""hello""#).unwrap();
+        let json_array_big = Json::from_str(r#"["a", "c"]"#).unwrap();
+        let json_array_small = Json::from_str(r#"["a", "b"]"#).unwrap();
+        let json_object = Json::from_str(r#"{"a": "b"}"#).unwrap();
+        let test_cases = vec![
```

Comment:
IMO it's better to group the test case in orders like what's in tidb:
1. values of same type
2. values of different types